### DISCUSSION TITLE
Ability to activate in existing worlds with Multiplayer Support, along with schematics for survival Minecraft.

### DISCUSSION THREAD

**Discussion by masonfaulk2008** [Mojang/minecraft-editor#44] [Feedback]
Add /gamemode editor command to base game. Map creation, is not only solo. The ability to invite your friends for play testing sake, and have access to the normal bedrock experience. Is very necessary in my opinion. 
Picture this:
You start up a world for map making, you type in /gamemode editor and get access to the UI. You build a build, but you want to detail it with your friends. You invite your friend to the game and go on a detailing adventure. Your friend suggests adding a new building, so you give them operator access, and access to editor mode (assuming they have a pc) you and your friends build an amazing map and publish it. 

This also gives access to the mode, for tinkering with on consoles, given that they have a keyboard and mouse. I hate the idea of it being so closed down. I get that this is made for map making, but imagine the memories you and you friends can make.

Content creation for Minecraft is un preventable, so embrace it. With Schematics, a player can design their dream base in editor/creative mode. And they will be able to save it as a schematic for survival on their other world. This will have a hologram of a build and a list of materials and where to place them for survival purposes. You grind out your resources and finally get to build it. This also gives players access to schematics from other players, allowing them to build to their hearts content.

**Comment by Landy005** [comment]
Schematics is a third party format, mojang uses mcstructures, although schematic support would be awesome indeed! Although this is referred to in https://github.com/Mojang/minecraft-editor/discussions/26

**Reply by masonfaulk2008** [reply]
Totally missed the point, I don’t want copy and paste, I want to design something in Editor mode, and export it as a schematic, to use in survival. It’d have a list of materials for me to mine and craft, and it’d have a halo graphic design of the base for example I could move around. Once I’m satisfied with the placement, there would be holographic blocks, with no collision that show me what to place.

**Reply by masonfaulk2008** [reply]
Along with that, you’d see that I posted this 5 hours ago at the time of writing, and your reply to #26 was 3 hours ago

**Reply by Landy005** [reply]
> Along with that, you’d see that I posted this 5 hours ago at the time of writing, and your reply to #26 was 3 hours ago

https://github.com/Mojang/minecraft-editor/discussions/26 was posted few hours before yours, I'm just referring to the fact that both bring up the subject of a sort of schematic to save/export. 



> Totally missed the point, I don’t want copy and paste, I want to design something in Editor mode, and export it as a schematic, to use in survival. It’d have a list of materials for me to mine and craft, and it’d have a halo graphic design of the base for example I could move around. Once I’m satisfied with the placement, there would be holographic blocks, with no collision that show me what to place.

What you're requesting is a sort of copy paste and is built on the base of copy pasting though... and I'm just sharing the fact schematics is a third party format, mojang has mcstructures to do holographic stuff with, I'm not sure they'd use the schematic format. 

I also think holographs in your survival world don't have much relatability to the editor in general, sounds more like addon work. Matter a fact there are addons that can already do this.
